user-registration-app/src/app/views/registration-form/registration-form.component.ts

```diff
@@ -109,8 +71,6 @@ export class RegistrationFormComponent implements OnInit {
         return 'Are you travelling from far away? You may be eligible for reimbursement!';
```

Comment:
Am I dumb or should 'Are you travelling from far away? ...' not have a case above it for travel reimbursement? Also travelling is spelled wrong. Also also do these actually appear? cause we should check that. And do we have a general error message saying that something went wrong and they should email technology@hackpsu.org?

Reply:
Do you think we should just delete the travel reimbursement field entirely or just not throw an error when it's blank?

Reply:
Nvm about the deleting thing Manik said he'd rather keep it, but are you saying we shouldn't throw an error

Reply:
yeah within the registration form we should keep it, but this return for the error has no case statement above it, right?

Reply:
Wait it has `case '.travelReimbursement':`, or do you mean something else

Reply:
On line 70 (the new diff)

Reply:
![image](https://user-images.githubusercontent.com/35966427/100673972-67ee8b80-3332-11eb-89fa-5b73fe8879e3.png)
The spelling is region dependant apparently

Reply:
These warnings appear when some of the required registration fields aren't filled out. When the submit button is pressed, the json validator is run and if there are any errors, this try catch will throw an alert for all applicable errors

Reply:
If something else goes wrong, this won't know, because all it does is check that the Registration object that's being created from their submission adheres to the Registration validation guards we set in registeredUserSchema.json

Reply:
> Wait it has `case '.travelReimbursement':`, or do you mean something else

...it does? I can't see it in the diffs for some reason lmaoo my b

Reply:
Lmao, since there were no changes above it, it doesn't show the file unless you click the expander on the left side